packages/autorest.typescript/test/rlcIntegration/generated/multipleInheritanceRest/samples-dev/getCatSample.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createMultipleInheritanceRestClient from "@msinternal/multiple-inheritance-rest";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to call operation GetCat
+ *
+ * @summary call operation GetCat
+ */
+async function getCatSample() {
+  const client = createMultipleInheritanceRestClient();
```

Comment:
There is no credential and endpoint information in the sample, is this expected?

Reply:
Yes this is our integration testing to test server and we have no credential required. And the endpoint info is set as default so it is also optional.

https://github.com/Azure/autorest.typescript/blob/27f6269db6e444b6054ad443860f7c569af4d316/packages/autorest.typescript/test/rlcIntegration/mediaTypesRest.spec.ts#L11